modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -109,12 +109,6 @@ spec:
       replicas: 1
       nodeSelector:
         node-role.kubernetes.io/worker: ""
-  curation:
```

Comment:
@sfortner-RH Do you want to review these changes to the logging topics in Dedicated?